thoth-app/src/component/publications_form.rs

```diff
@@ -358,6 +429,36 @@ impl Component for PublicationsFormComponent {
                                     oninput=self.link.callback(|e: InputData| Msg::ChangeIsbn(e.value))
                                     deactivated=isbn_deactivated
                                 />
+                                <label class="checkbox">
```

Comment:
I think this side of the form should not be displayed at all for digital publications (instead of disabling it)

Reply:
Agreed - have also implemented for PublicationComponent.